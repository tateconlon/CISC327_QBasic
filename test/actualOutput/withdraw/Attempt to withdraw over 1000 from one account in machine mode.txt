login as machine successful
Account 1234567 does not exist
transaction code 100000 invalid
Account 1234567 does not exist
transaction code 100000 invalid
Account 1234567 does not exist
transaction code 100001 invalid
Transaction summary written and system logged out successfully