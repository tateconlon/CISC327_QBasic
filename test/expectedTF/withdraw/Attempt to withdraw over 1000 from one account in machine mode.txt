DEP 1234567 100000 *** ***
DEP 1234567 100000 *** ***
EOS