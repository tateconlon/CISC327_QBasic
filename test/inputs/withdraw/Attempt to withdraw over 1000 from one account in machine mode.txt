login
machine
deposit
1234567
100000
deposit
1234567
100000
withdraw
1234567
100001
logout