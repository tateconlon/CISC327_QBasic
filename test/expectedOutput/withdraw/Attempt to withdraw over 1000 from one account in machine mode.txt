login as machine successful
deposit 1234567 100000 successful
deposit 1234567 100000 successful
cannot withdraw more than $1000.00 in a single transaction in machine mode
Transaction summary written and system logged out successfully